packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -257,9 +318,102 @@ class RenderParagraph extends RenderBox {
     return _textPainter.computeDistanceToActualBaseline(baseline);
   }
 
+  /// Intrinsics cannot be calculated without a full layout for
+  /// alignments that require the baseline (baseline, aboveBaseline,
+  /// belowBaseline)
+  bool _canComputeIntrinsics() {
+    for (PlaceholderSpan span in _placeholderSpans) {
```

Comment:
let's only do this in debug mode (call this _debugCanComputeIntrinsics, put the body mostly in an assert, etc)
in the methods that call this today, just assert it, and don't worry about what happens in release builds.

Reply:
https://docs.flutter.io/flutter/rendering/RenderBox/computeMinIntrinsicWidth.html says that we should be returning 0.0 for non-debug mode.

Reply:
my bad, carry on!